### PR TITLE
Fixed Youtube Extractor error 

### PR DESCRIPTION
**Pull request by dev-shashwatt**
This update addresses instability issues caused by the previous YouTube extractor. I have replaced the existing implementation in main.js with the new YoutubeiExtractor, which is more reliable. Users will need to run yarn install to install the new extractor before starting the bot as usual. This change aims to enhance performance and improve user experience.

### CODE REVIEW

File: CHANGES DONE.txt
Comment (revot334) [inline CHANGES DONE.txt:1]: remove this file and add the command to the right place in the readme (aka in the setup)

File: .env.example
Comment (revot334) [inline .env.example:1]: add back

File: main.js
Comment (revot334) [inline main.js:1]: looks great
also add new package to the package.json

**Review by revot334 — CHANGES_REQUESTED** [review]
Please review feed back

File: CHANGES DONE.txt
Comment (dev-shashwatt) [reply CHANGES DONE.txt:1]: okay i added it in the package.json so that it gets auto installed with the other packages when the user does yarn install

**Review by revot334 — APPROVED** [review]

**Pull request merged**